include/exadg/grid/mapping_deformation_structure.h

```diff
@@ -141,8 +145,12 @@ class DeformedMapping : public DeformedMappingBase<dim, Number>
       pde_operator->initialize_dof_vector(rhs);
       pde_operator->compute_rhs_linear(rhs, time);
 
-      auto const iter = pde_operator->solve_linear(
-        displacement, rhs, 0.0 /* no mass term */, time, false /* do not update preconditioner */);
+      auto const iter = pde_operator->solve_linear(displacement,
+                                                   rhs,
+                                                   0.0 /* no mass terms */,
+                                                   0.0 /* no mass terms */,
```

Comment:
I think these comments should be changed for the same reason as described here https://github.com/exadg/exadg/pull/475/files#r1282039088. The comments should be something like "no acceleration term" and "no damping term". Note that there are several other occurrences of this type in the code.